_posts/2019-07-07-project-courses.md

```diff
@@ -1,95 +1,64 @@
+Although our project courses seem to be effective, a caveat to consider before implementing similar courses at your institution is that our students are exposed to repeated practice using version control via git and GitHub before their first project course (see [this post](https://ubc-mds.github.io/2017-08-24-teaching-with-github/) on how we use these tools in MDS). Alternative collaboration platforms that might replace GitHub, especially for courses based on Jupyter notebooks, are [CoCalc](https://cocalc.com/), [Google Colab](https://colab.research.google.com/), [Kaggle Kernels](https://www.kaggle.com/kernels) or [RStudio Server Pro](https://www.rstudio.com/products/rstudio-server-pro/). These tools could be combined with some simple GitHub features such as issues for collaborative communication and use the GitHub website upload tool to upload a version of the project for peer review session and the final submission.
 
-## Conclusion
+Overall, from our experience with the UBC MDS program, I believe that project-based courses are necessary for a well-rounded data science education. These courses give students the opportunity to work in more real-world situations where the "right" answer is less well-defined, collaboration is required and many aspects of data science are brought together. We also believe that students should have the opportunity to do more than one project in their data science program so that they can repeatedly practice an entire project from beginning to end in a lower stakes environment before they are required to do so on the job. 
 
-From our experience with the University of British Columbia's Master of Data Science program we have come to the opinion that project-based courses are necessary for a well-rounded Data Science education. We believe this is because it gives the students to work in a more real-life situation where the "right" answer is less well defined, collaboration is required and many axes of Data Science are brought together. We also believe that students should have the opportunity to do more than 1 project in their Data Science program so that they get to repeatedly practice an entire project from beginning to end in a lower stakes environment before they are required to do so on the job. 
+--------
+[Tiffany Timbers](https://www.tiffanytimbers.com/) is Option Co-Director of the MDS Vancouver program and an Instructor in the UBC Department of Computer Science.
```

Comment:
```suggestion
[Tiffany Timbers](https://www.tiffanytimbers.com/) is Option Co-Director of the MDS Vancouver program and an Instructor in the UBC Department of Statistics.
```

Reply:
I'm in the Stat dept @mgelbart 😉

Reply:
Also, given your heroic effort in editing, should you also be listed as an author on this post?